Asterisk.2013/Asterisk.NET/Manager/ManagerConnection.cs

```diff
@@ -595,11 +600,13 @@ public ManagerConnection()
             Helper.RegisterEventHandler(registeredEventHandlers, 90, typeof(BridgeEnterEvent));
             Helper.RegisterEventHandler(registeredEventHandlers, 91, typeof(BridgeLeaveEvent));
             Helper.RegisterEventHandler(registeredEventHandlers, 92, typeof(BlindTransferEvent));
-            
 
-			#endregion
+            Helper.RegisterEventHandler(registeredEventHandlers, 93, typeof(DTMFBeginEvent));

```

Comment:
This will conflict with another PR currently in review. This is a pain and I hate this way that events are managed, but for now could you bump the IDs into the 100's to avoid you both having to deal with conflict depending on who gets merged first.